lib/jnpr/jsnapy/check.py

```diff
@@ -75,6 +75,27 @@ def get_err_mssg(self, path, ele_list):
         """
         This function generates error message, if nothing is given then it will generate default error message
         """
+        temp = ['err', 'info', 'ignore-null']
+        value_list = []
+        for key, value in path.items():
+            if key not in temp:
+                if ',' in value:
+                    value_list = [elements.strip() for elements in value.split(',')]
+        if len(value_list) > 0: 
+            temp_str = '$1'
```

Comment:
Problems with this code:
- It will not work for arguments greater than 2, i.e $3 will fail. Functionality can be useful in `is-in` operator.
- You should use regex match and replace so that minimal code is required.
- Please refer to dict.get usage [here](https://www.tutorialspoint.com/python/dictionary_get.htm). No need to have the default string every-time as you are not going to find anything that matches `\$\d+`, as you have hard-coded it.